src/MiniProfiler.Shared/Data/ProfiledDbCommand.cs

```diff
@@ -196,6 +196,12 @@ public override UpdateRowSource UpdatedRowSource
             set => _command.UpdatedRowSource = value;
         }
 
+        /// <summary>
+        /// Creates a wrapper data reader for <see cref="ExecuteDbDataReader"/> and <see cref="ExecuteDbDataReaderAsync"/> />
+        /// </summary>
+        protected virtual DbDataReader CreateDataReader(DbDataReader original, IDbProfiler profiler)
```

Comment:
Let's align the naming here - we have `CreateDbCommand` with Db elsewhere and `CreateDataReader` here - can you please change this to `CreateDbDataReader` (more correct given the return type).